testutils/testutils.go

```diff
@@ -30,6 +30,7 @@ var (
 )
 
 func init() {
+	verbose = flag.Bool("vv", false, "Verbose Logging?")
```

Comment:
[NIT] Let's drop the question mark:

```suggestion
	verbose = flag.Bool("vv", false, "Verbose logging.")
```

Reply:
@mattayes that is a super NIT.   We're talking about logging in tests.